CHANGELOG.rst

```diff
@@ -6,6 +6,8 @@ in development
 
 Fixed
 ~~~~~
+* Fix issue #5983
```

Comment:
Can we include a more descriptive changelog for this fix?
What's the bug and what it fixes would be helpful.

Reply:
All right, will add it right away

Reply:
@armab 
I have changed the description in the PR and the CHANGELOG.rst as well.